doc/whats_new/v0.22.rst

```diff
@@ -160,6 +160,10 @@ Changelog
   underlying :class:`linear_model.LassoLars` when `algorithm='lasso_lars'`.
   :issue:`12650` by `Adrin Jalali`_.
 
+- |Fix| :func:`decomposition.nmf._special_sparse_dot()` now uses batching to avoid
```

Comment:
`_special_sparse_dot` is a private function, I would prefer not mentioning it:
```rst
:class:`decomposition.NMF(solver='mu')` fitted on sparse input matrices now uses batching...
```

Reply:
I think this should be regarded as Efficiency rather than Fix